Makefile

```
@@ -243,6 +244,7 @@ argoexec-image:
 %-image:
 	[ ! -e dist/$* ] || mv dist/$* .
 	docker buildx build \
+		--platform $(TARGETPLATFORM) \
```

```suggestion
		--platform $(TARGET_PLATFORM) \
```

same as above regarding separating words. otherwise, this line seems correct